scipy/optimize/_qap_umeyama.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+from . import linear_sum_assignment, OptimizeResult
+
+
+def eigenvectors(x):
```

Comment:
This function looks too small to refactor I think. In fact this file is tiny to get separated for my taste but it's not a blocker.

Reply:
I'll put it back in `_qap.py`.